packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -35,7 +35,8 @@ export const {
 
 export const enableProfilerTimer = __PROFILE__;
 export const enableProfilerCommitHooks = __PROFILE__;
-export const enableProfilerNestedUpdatePhase = __PROFILE__;
+export const enableProfilerNestedUpdatePhase =
+  __PROFILE__ && dynamicFeatureFlags.enableSchedulingProfiler;
```

Comment:
The flag will be dynamic for the profiling build only. It will still be stripped out of the production build.